src/LibraryManager.Vsix/Commands/CommandTable/LocalizationFiles/VSCommandTable.cs.xlf

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd" version="1.2">
-  <file original="E:\LibraryManager\src\LibraryManager.Vsix\Commands\CommandTable\VSCommandTable.en.vsct" source-language="en" target-language="cs" datatype="xml">
+  <file original="C:\Users\bhsubra\Desktop\LibraryManager\src\LibraryManager.Vsix\Commands\CommandTable\VSCommandTable.en.vsct" source-language="en" target-language="cs" datatype="xml">
```

Comment:
This is the best comment I found - https://stackoverflow.com/questions/15071272/what-should-i-use-as-the-xliff-original-file-in-symfony-2

The spec doesn't say much and actually marks it as optional. I think it's mostly tool-dependent usage. We probably should switch vsct to xliff to produce a relative path or GitHub URL so that we avoid meaningless diffs like this (since each dev will likely have the file in a different location)

Reply:
The spec link - http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#original